src/components/Banner/Banner.scss

```diff
@@ -116,69 +116,131 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tight);
   --p-rgb-text: 33, 43, 54;
   position: relative;
   display: flex;
+
+  // stylelint-disable selector-max-class, selector-max-combinators, selector-max-specificity
+  &.statusCritical .PrimaryAction .Button {
```

Comment:
Nitpick: Do these status classes need to be nested inside `.Banner`? Could they live at the top level instead?

(only comment, everything else looks great)